src/main/java/no/digipost/signature/client/portal/PortalJob.java

```diff
@@ -143,11 +143,6 @@ public Builder availableFor(long duration, TimeUnit unit) {
             return this;
         }
 
-        public Builder withNonsensitiveTitle(String nonsensitiveTitle) {
-            target.nonsensitiveTitle = Optional.of(nonsensitiveTitle);
-            return this;
-        }
-
```

Comment:
Skal man ikke fortsatt kunne legge til en `nonsensitiveTitle` for portaljobber?

Reply:
Jo, det skal man vel, men metoden ble ikke brukt. Skal den være der likevel?

Reply:
Ja, det betyr bare at den ikke blir brukt av noen tester. Den må fortsatt være der for at de som bruker biblioteket skal kunne bruke den 😊